pkg/distro/defs/fedora/distro.yaml

```diff
@@ -167,15 +167,18 @@
     additional_dracut_modules:
       - "net-lib"
     squashfs_rootfs: true
-    condition:
-      version_less_than:
-        "42":
-          # config is fully replaced
+    conditions:
+      "f41 uses ifcfg in dract but already a squashfs rootfs":
```

Comment:
'but already' is confusing. I'd word it like "f41 dracut requires ifcfg, rootfs is squashfs"

Reply:
I will address those in a followup